src/scss/_standard.scss

```diff
@@ -169,23 +168,27 @@
 	float: left;
 	width: $o-card-image-left-width;
 	padding-right: $o-card-spacing-unit;
+	margin: 0;
 }
 
 /// Styles for a right aligned image
 @mixin oCardImageRight {
 	float: right;
 	width: $o-card-image-right-width;
 	padding-left: $o-card-spacing-unit;
+	margin: 0;
 }
 
 /// Styles to force an image to the top of the card
 @mixin oCardImageTop {
 	order: -1;
+	margin: 0 0 oGridGutter();
```

Comment:
There's no dependency for `o-grid` at the moment, I don't think this will work as it is, on it's own? (same with line 193)